CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Compatible changes
 
--
+- Allow to override humanization for inherited assignable_values
```

Comment:
Remember to extend the `Humanized labels` section in the README so users can learn how I18n lookup works with inheritance.